cPP_DSC.adoc

```diff
@@ -4229,34 +4407,42 @@ FMT_SMR.1
 
 |FCS_RBG.2 
 |FCS_RBG.1 
-|FCS_RBG.1 is required by this PP.
+|FCS_RBG.1 - included
 
 |FCS_RBG.3 
 |FCS_RBG.1 
-|FCS_RBG.1 is required by this PP.
+|FCS_RBG.1 - included
 
 |FCS_RBG.4 
 |FCS_RBG.1
 
 FCS_RBG.5 
-|FCS_RBG.1 is required by this PP. FCS_RBG.5 is required when combining multiple entropy sources.
+|FCS_RBG.1 - included
+
+FCS_RBG.5 - (selection-based SFR) included
 
 |FCS_RBG.5
 |FCS_RBG.1
 
 [FCS_RBG.2 or FCS_RBG.3 or RCS_RBG.4]
-|FCS_RBG.1 is required by this PP. A second source is required for combining sources of entropy.
+|FCS_RBG.1 - included
+
+FCS_RBG.2 - (selection-based SFR) included
+
+FCS_RBG.3 - (selection-based SFR) included
+
+FCS_RBG.4 - (selection-based SFR) included
```

Comment:
All of these (except FCS_RBG.1) are optional, not selection-based

Reply:
fixed